CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ If you are looking for developer-facing changes, check out [CHANGELOG-API.md](./
 
 <!-- next version -->
 
+## v0.91.0
+
+### 🛑 Breaking changes 🛑
+
+### 🚀 New components 🚀
+
+### 💡 Enhancements 💡
+
+- `awsproxyextension`: Expose service_name as configurable option. Previously, it was hardcoded to xray 
+
+### 🧰 Bug fixes 🧰
```

Comment:
please don't edit CHANGELOG.md, please run `make chlog-new` from the root of the repository and fill in the file created.

Reply:
Fixed